skills_ml/algorithms/corpus_creators/basic.py

```diff
@@ -90,11 +90,12 @@ class Doc2VecGensimCorpusCreator(CorpusCreator):
     ]
     join_spaces = ' '.join
 
-    def __init__(self, generator=None):
+    def __init__(self, generator=None, occ_classes=[]):
```

Comment:
I'd like to see the docstring updated with some information about this new functionality. It already is lacking some information about what the passed-in arguments do, but adding another argument with new functionality makes it even more important to update.